### PR TITLE
chore: update context [DIA-50447]

### PR DESCRIPTION
**Pull request by sotran**
## Description

Update CCI Contexts to get rid of the deprecated org-global-v2

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ workflows:
           name: 🚀👩‍💻🇨🇦 deploy-dev-ca2
           stage: dev
           place: ca2
-          context: org-global-v2
+          context: static-cdn-deployer-dev-ca2
```
Comment (mbelang) [inline .circleci/config.yml:94]:
```suggestion
          context: static-cdn-deployer-dev-ca
```

Comment (mbelang) [reply .circleci/config.yml:94]: and while your are at it, change the name of the deploy job to dev-ca as well

Comment (arththebird) [reply .circleci/config.yml:94]: We'll probably need to change the `place: ca2` to `place: ca` as well for all occurence of `ca2`

File: .circleci/config.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ workflows:
           stage: dev
           place: ca2
```
Comment (arththebird) [inline .circleci/config.yml:79]:
```suggestion
          place: ca
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -78,7 +78,7 @@ workflows:
...
           ephemeral: yes
-          context: org-global-v2
+          context: static-cdn-deployer-dev-ca2
```
Comment (arththebird) [inline .circleci/config.yml:81]:
```suggestion
          context: static-cdn-deployer-dev-ca
```

**Comment by aloeci** [comment]
<h2> 🚀🥀 Ephemeral Deployment URL</h2>

 https://covid-19.dev.dialogue.co/branches/chore/dia-50447/context/index.html 

 Note: the url will expire on Mon 23 Jan 2023 04:40:30 PM UTC

**Review by mbelang — APPROVED** [review]
Please add a description

**Pull request merged**